.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```
@@ -96,7 +96,12 @@ steps:
   - script: |
       echo "Run Cilium Connectivity Tests"
       cilium status
-      cilium connectivity test --connect-timeout 4s --request-timeout 30s --test '!pod-to-pod-encryption,!node-to-node-encryption'
+      if [ "$CILIUM_VERSION_TAG" = "cilium-nightly-pipeline" ]
+      then
```

nit: move `then` to the same line..?
```suggestion
      if [ "$CILIUM_VERSION_TAG" = "cilium-nightly-pipeline" ]; then
```